clang/test/SemaSYCL/unnamed-kernel.cpp

```diff
@@ -113,7 +113,7 @@ struct MyWrapper {
 int main() {
   cl::sycl::queue q;
 #ifndef __SYCL_UNNAMED_LAMBDA__
-  // expected-error@Inputs/sycl.hpp:233 {{unnamed lambda '(lambda at /iusers/sundaram/forked-repo/llvm/clang/test/SemaSYCL/unnamed-kernel.cpp:119:54)' used}}
+  // expected-error-re@Inputs/sycl.hpp:233 {{unnamed lambda '(lambda at {{.*}}unnamed-kernel.cpp{{.*}}' used}}
```

Comment:
I would have preferred if the filename was also reg-exped away, so the test is independent of its name, but I am okay with this change.

Have you had a chance to look at the pre-Jenkins failures?

Reply:
One suggestion that I'd made (and filed a bug report for) is to not have the filename/line number there at ALL, and instead use the marker syntax for all the diagnostic locations that appear inside those headers.

Reply:
See https://github.com/intel/llvm/issues/3415.